requirements.txt

```diff
@@ -2,7 +2,7 @@
 ipython>=5.5.0
 ipywidgets>=7.6.5  # required by pycaret.internal.display
 tqdm>=4.62.0  # required by pycaret.internal.display
-numpy>=1.21, <1.24  # Can't >=1.24 because of np.float deprecation
+numpy>=1.21, <1.27
```

Comment:
We can't see where the code base uses `np.float`. Only `np.float32` and `np.float64` have been discovered so far. So, we think it will be good to go.